src/TropicalGeometry/numbers.jl

```diff
@@ -0,0 +1,449 @@
+# module Tropical
+
+export tropical_numbers,
+       @tropical,
+       determinant,
```

Comment:
but it's called `det` everywhere else... if we do this, then I think it should at least be an alias, not just an arbitrary subset of all possible determinant methods. So:
```
const determinant = det
```

Reply:
I renamed it to `det` in order to have some progress.